c2pa_c_ffi/src/c2pa_stream.rs

```diff
@@ -108,6 +108,17 @@ impl C2paStream {
         let context_ptr = std::mem::replace(&mut self.context, std::ptr::null_mut());
         unsafe { Box::from_raw(context_ptr) }
     }
+
+    /// Checks the stream context.
+    fn check_context(&self) -> std::io::Result<()> {
```

Comment:
I don't think I have a rule that all stream implementations have to set a non-null context. They don't technically need to use a context, but in practice they all might need to. If we are ok with insisting that they must supply some kind of non-null context, this is ok.

Reply:
Removing for now then.